internal/storage/storage.go

```diff
@@ -40,7 +40,7 @@ type Storage struct {
 	Namespaces []*Namespace
 }
 
-// DecryptV1 tries to decrypt the original unsecure SHA1 storage.
```

Comment:
this wasn't actually a typo, thats literally what the function is named

https://github.com/yitsushi/totp-cli/blob/main/internal/security/unsecure.go

(Whether that function name was a typo is another question)

Reply:
That's true, and with the new wording it implies the storage itself was SHA1 "encrypted" (which is not an encryption, it's a hashing algo). But in all cases, this comment is not valid at all. It does not try to "decrypt the original unsecure SHA1 storage", it tries to decrypt using the old method (`UnsecureSHA1`) of hashish of the password. :/

Reply:
This should make it a bit more clear: https://github.com/yitsushi/totp-cli/pull/111